guide/C/ch_invest.xml

```diff
@@ -1744,7 +1744,7 @@ Income
             <row>
               <entry>Assets:Brokerage Account:Stock:IBM</entry>
 
-              <entry>-51.7598</entry>
+              <entry>&mdash;51.7598</entry>
 
               <entry>90.00</entry>
 
```

Comment:
Everything above this line should really be in a separate commit. It's more housekeeping and doesn't really have anything to do with the bug.